changelog/db.changelog-1.54.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Sam Plackett" id="1">
+    <tagDatabase tag="v1.54.0" />
+  </changeSet>
+  <changeSet author="Sam Plackett" id="2">
+    <createTable tableName="frnAgreementClosed">
+      <column name="closedId" type="INTEGER">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="closedId_pkey" />
+      </column>
+      <column name="schemeId" type="INTEGER" />
+      <column name="frn" type="BIGINT" />
+      <column name="agreementNumber" type="VARCHAR(50)" />
+      <column name="closureDate" type="DATE" />
```

Comment:
Think this has to be type `TIMESTAMP` rather than `DATE`?

Reply:
changed